### ISSUE TITLE
[C++] Line continuation character ignored by syntax highlighting

### ISSUE THREAD

**Issue by septillion-git** [bobbylight/RSyntaxTextArea#225]
The line continuation character \ is not parsed by the syntax highlighting if it's part of a line comment.
```C++
// I start a comment on this line but I end with the line continuation character \
byte test = 5; //This WHOLE line is a comment because of the line continuation character on the last line

//The byte test is thereby NOT declared but the syntax highlighting of the IDE does not display it as comment
```
The Markdown syntax highlighting does parse it correct :)

This is causing trouble in Arduino for newbies. They have no knowladge about the line continuation character which can cause them trouble but the highlighting isn't pointing it out either. If the line was highlighted as comment the error would have been more clear.

Link to a case where it happened:
http://forum.arduino.cc/index.php?topic=449738.0

Issue on Arduino Git arduino/Arduino#5888

**Comment by bobbylight** [comment]
Thanks for this! I'll squeeze this into the next release.

**Comment by bobbylight** [comment]
Feel free to try this fix out early with the latest 2.6.1-SNAPSHOT available on [SonaType](https://oss.sonatype.org/content/repositories/snapshots/com/fifesoft/rsyntaxtextarea/).  I hope to release 2.6.1 to Maven Central in the next week or so if you'd rather wait for that.
